nms/app/packages/magmalte/app/views/equipment/GatewayDetailConfig.js

```diff
@@ -339,14 +339,23 @@ function GatewayRAN({gwInfo}: {gwInfo: lte_gateway}) {
   const enbInfo =
     gwInfo.connected_enodeb_serials?.reduce(
       (enbs: {[string]: EnodebInfo}, serial: string) => {
-        if (enbCtx.state.enbInfo[serial] != null) {
+        if (enbCtx?.state?.enbInfo?.[serial] != null) {
           enbs[serial] = enbCtx.state.enbInfo[serial];
         }
         return enbs;
       },
       {},
     ) || {};
+  const dhcpServiceSts = gwInfo.cellular.dns?.dhcp_server_enabled ?? true;
```

Comment:
I'm just checking that the default state should actually be true. Seems weird that it would be so if this flag didn't exist.

Reply:
Yes. It does seem a little weird. @ardzoht do you know why we don't have dns configuration by default and set the dhcp_service_enabled as true or instead call the flag as dhcp_service_disabled?